### PR TITLE
Fix deprecated messages

### PR DESCRIPTION
**Pull request by nhducit**

### CODE REVIEW

**Comment by nhducit** [comment]
ping @chenglou @jsdf

File: src/CommentList.re [hunk 1]
```diff
@@ -23,19 +23,19 @@ let make = (~story: StoryData.story_with_comments, _children) => {
       };
     | None => collapsed
     };
-  let getCommentIdFromEvent = (event: ReactEventRe.Mouse.t) =>
```
Comment (nhducit) [inline src/CommentList.re]: I did not made these changes 🤔

Comment (nhducit) [reply src/CommentList.re]: these changes are created when I edit `CommentList.re` file and hit `ctrl + S`

Comment (nhducit) [reply src/CommentList.re]: Not sure whether they are expected changes

Comment (chenglou) [reply src/CommentList.re]: I hope you're using https://github.com/chenglou/upgrade-reason-react and not upgrading manually btw lol. Yes, `ReactEventRe` is deprecated.

**Comment by chenglou** [comment]
Thanks!

File: src/CommentList.re (hunk 1 above)
Comment (nhducit) [reply src/CommentList.re]: I use  chenglou/upgrade-reason-react to migrate in the previous PR, not sure why it still cause this issue

Comment (nhducit) [reply src/CommentList.re]: https://github.com/reasonml-community/reason-react-hacker-news/pull/18

Comment (chenglou) [reply src/CommentList.re]: Ah weird, maybe I didn't catch this case. Thanks.

Comment (nhducit) [reply src/CommentList.re]: you're welcome

**Pull request merged**